omod/src/main/webapp/resources/scripts/taggedObjects.js

```diff
@@ -0,0 +1,42 @@
+
+var app = angular.module('taggedObjects',['personService','tagService']);
+
+app.config(['$locationProvider', function($locationProvider){
+    $locationProvider.html5Mode({
+        enabled: true,
+        requireBase: false
+    });
+}]);
+
+app.controller('taggedObjectsCtrl',['$scope','$location','Person','TagService', function ($scope,$location,Person,TagService) {
+
+    var tags = [];
+    $scope.init = function () {
```

Comment:
I don't think you need the init function, this can all go in the main function you are passing to the controller